client/public/index.html

```diff
@@ -24,7 +24,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    
+    <style>
+      html, body, #root, #root>div {
```

Comment:
This is not the right way. Instead give `className` or `Id` to the `div` tag and then give required styles in `.css` file.